opencti-platform/opencti-front/src/private/components/common/lists/FilterDate.tsx

```diff
@@ -70,15 +70,17 @@ const FilterDate: FunctionComponent<FilterDateProps> = ({
       value={findFilterFromKey(inputValues, filterKey, operator)?.values[0] || null}
       onChange={(value) => handleChangeDate(value as Date)}
       onAccept={(value) => handleAcceptDate(value as Date)}
-      renderInput={(params) => (
-        <TextField
-          variant="outlined"
-          size="small"
-          fullWidth={true}
-          onKeyDown={(event) => handleValidateDate(event)}
-          {...params}
-        />
-      )}
+      slotProps={{
+        textField: (params) => (
+          <TextField
+            variant="outlined"
+            size="small"
+            fullWidth={true}
+            onKeyDown={(event) => handleValidateDate(event)}
+            {...params}
+          />
+        ),
```

Comment:
```suggestion
        textField: (params) => ({
          ...params,
          size: 'small',
          variant: 'outlined',
          fullWidth: true,
          onKeyDown: (event) => handleValidateDate(event),
        }),
```

Reply:
![image](https://github.com/OpenCTI-Platform/opencti/assets/18297909/0a6e007e-48ee-40dc-87f5-73e17025f71f)

Reply:
Thank you so much it has been very helpful !